fe1-web/src/features/wallet/screens/WalletHome.tsx

```diff
@@ -1,49 +1,114 @@
 import { useNavigation } from '@react-navigation/native';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { StyleSheet, View, ViewStyle } from 'react-native';
+import { useSelector } from 'react-redux';
 
-import { TextBlock, WideButtonView } from 'core/components';
+import { QRCode, TextBlock, WideButtonView } from 'core/components';
 import containerStyles from 'core/styles/stylesheets/containerStyles';
+import { LaoEventType } from 'features/events/objects';
+import { makeEventByTypeSelector } from 'features/events/reducer';
+import { selectCurrentLao } from 'features/lao/reducer';
+import { RollCall } from 'features/rollCall/objects';
 import STRINGS from 'resources/strings';
 
-import { WalletStore } from '../store';
+import { RollCallTokensDropDown } from '../components';
+import * as Wallet from '../objects';
+import { createDummyWalletState, clearDummyWalletState } from '../objects/DummyWallet';
+import { RollCallToken } from '../objects/RollCallToken';
 
 const styles = StyleSheet.create({
+  homeContainer: {
+    ...containerStyles.centeredXY,
+    padding: '30px',
```

Comment:
```suggestion
    padding: 30,
```
I think both work, but we're not used to this notation in the project.